docs/api/README.md

```diff
@@ -192,3 +192,46 @@ MatchMediaBase.init({
 ## MatchMediaItem
 
 The match media item mixin applies media queries set in `MatchMediaBase` to a component. To use, add it as a mixin to any component that is a descendent of the component with `MatchMediaBase` that should include media query styles.
+
+## Style Component
+
+The `<Style>` component renders an HTML `<style>` tag containing a set of CSS rules. Using it, you can define an optional `scopeSelector` that all selectors in the resulting `<style>` element will include.
+
```

Comment:
OK, did my back reading -- http://facebook.github.io/react/docs/jsx-in-depth.html#html-tags-vs.-react-components -- we're fine ;)